### PR TITLE
add dbt-compile tip for docs only job

### PR DESCRIPTION
**Pull request by mirnawong1**
## Description & motivation

Adjusted docs generate page to add tip for docs-only job per slack thread here: https://dbt-labs.slack.com/archives/C017GDLAF7D/p1666730266659529

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by mirnawong1** [comment]
hey @kbrock91 and @bennieregenold7 - can you have a look when you have a moment? also is there any recommendations we want to include here?

File: website/docs/docs/collaborate/cloud-build-and-view-your-docs.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ To set up a job to generate docs:
 3. Under "Execution Settings," select **Generate docs on run**.
    <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/documentation-job-execution-settings.png" title="Setting up a job to generate documentation"/>
 
+:::tip
```
Comment (bennieregenold7) [inline website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]: I think this needs a title. I was able to get this working with the block below

```
:::tip Use dbt compile

To create and schedule a documentation-only job at the end of your production jobs, add the `dbt compile` command under **Commands**. 

:::
```

Comment (mirnawong1) [reply website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]: good callout @bennieregenold7

**Review by bennieregenold7 — DISMISSED** [review]

File: website/docs/docs/collaborate/cloud-build-and-view-your-docs.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]: thank you again! i've tweaked it a bit so it flows after the steps and changed the title so it's more catered to those that want to run a docs-only job. maybe it was me, but using the 'use dbt compile' title made it read like it was a mandatory step for ppl?

**Review by bennieregenold7 — DISMISSED** [review]

File: website/docs/docs/collaborate/cloud-build-and-view-your-docs.md [hunk 2]
```diff
@@ -19,6 +19,12 @@ To set up a job to generate docs:
 
 4. Click **Save**. Proceed to [configure project documentation](#configure-project-documentation) so your project generates the documentation when this job runs.
 
+:::tip Tip &mdash; For documentation-only jobs
```
Comment (matthewshaver) [inline website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]:
```suggestion
:::tip Tip &mdash; Documentation-only jobs 
```

Comment (matthewshaver) [reply website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]: Love this idea - been getting feedback that these kinds of hints would be more welcome in our docs.

File: website/docs/docs/collaborate/cloud-build-and-view-your-docs.md [hunk 3]
```diff
@@ -19,6 +19,12 @@ To set up a job to generate docs:
 
 4. Click **Save**. Proceed to [configure project documentation](#configure-project-documentation) so your project generates the documentation when this job runs.
 
+:::tip Tip &mdash; For documentation-only jobs 
+
+To create and schedule a documentation-only job at the end of your production jobs, add the `dbt compile` command in the **Commands** section.
```
Comment (matthewshaver) [inline website/docs/docs/collaborate/cloud-build-and-view-your-docs.md]:
```suggestion
To create and schedule documentation-only jobs at the end of your production jobs, add the `dbt compile` command in the **Commands** section.
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**